CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 For official release notes for Docker Engine CE and Docker Engine EE, visit the
 [release notes page](https://docs.docker.com/engine/release-notes/).
 
+## 18.09.6 (2019-05-02)
+
+### Builder
+
+- Fix `COPY` and `ADD` with multiple `<src>` do not invalidate cache if `DOCKER_BUILDKIT=1`
```

Comment:
If we want to link to something; this is the issue; https://github.com/moby/moby/issues/38964 / https://github.com/moby/buildkit/issues/853

and these are the PR's https://github.com/moby/buildkit/pull/858 / https://github.com/moby/buildkit/pull/911